src/main/scala/io/viash/Main.scala

```diff
@@ -403,14 +402,14 @@ object Main extends Logging {
         Some(ViashNamespace.targetOutputPath(
           targetDir = td,
           runnerId = ex.id,
-          namespace = appliedConfig.config.functionality.namespace,
-          functionalityName = appliedConfig.config.functionality.name
+          namespace = appliedConfig.config.namespace,
+          componentName = appliedConfig.config.name
```

Comment:
could be renamed to simply `name`, or not?

Reply:
Sure!